aiokafka/coordinator/assignors/abstract.py

```diff
@@ -37,8 +51,9 @@ def metadata(self, topics):
             MemberMetadata struct
         """
 
+    @classmethod
```

Comment:
(offtopic) Why is it class method? It's strange to have a shared state among all clients